README.md

```diff
@@ -131,3 +169,21 @@ It seems that usually recipes require several things to be tested:
 
 * That recipe flow is working as expected
 * Timeouts: reproduce every timeout with meaningful values (timeout 0.5s and block for 0.6s)
+
+
+### Run some tests directly
+
+Another way to run tests only which I am interested in. Or this is useful when
```

Comment:
@rhdxmr probably it will sound better if we change `which I am interested in` => `which you are interested in`

Reply:
@cybergrind  Thanks. I applied your correction 🙂